subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/ForkCapableRelocationIntegrationTest.groovy

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures
+
+import org.gradle.test.fixtures.file.TestFile
+
+
+abstract class ForkCapableRelocationIntegrationTest extends AbstractProjectRelocationIntegrationTest {
+
+    abstract String getDaemonConfiguration()
+
+    abstract String getForkOptionsObject()
+
+    String getDaemonTask() {
+        return taskName.split(':').last()
+    }
+
+    @ToBeFixedForConfigurationCache(bottomSpecs = ["ScalaCompileRelocationIntegrationTest"])
```

Comment:
Why does scala this not work with configuration caching?

Reply:
It has something to do with the `IncrementalCompileOptions` being nulled out when rehydrated from the configuration cache.  This is not new with these changes - the tests in the parent class are marked as `ToBeFixedForConfigurationCache` for the scala subclass and most of the scala compilation integration tests are marked this way and fail in the same manner as well.